lib/workers/repository/updates/generate.ts

```diff
@@ -196,23 +193,13 @@ export function generateBranchConfig(
       );
       throw new Error(CONFIG_SECRETS_EXPOSED);
     }
-    upgrade.commitMessage = upgrade.commitMessage.trim(); // Trim exterior whitespace
-    upgrade.commitMessage = upgrade.commitMessage.replace(regEx(/\s+/g), ' '); // Trim extra whitespace inside string
     upgrade.commitMessage = upgrade.commitMessage.replace(
       regEx(/to vv(\d)/),
       'to v$1'
     );
```

Comment:
Wont this be lost when `commitBody` isa set now?

Reply:
Sorry, I did not get the question. Could you please elaborate?

Reply:
@viceice ?

Reply:
please follow code flow. it looks like this code is no longer always applied

Reply:
I have added new unit test, seems like everything works fine. Meanwhile I am not sure this functionality is still necessary

Reply:
this will happen, when the `newValue` has a leading `v` like it's often used for git tags

Reply:
You don't have a test with `commitBody` and `vv` `commitMessage`.

I think this code now needs to be moved below the `if (upgrade.commitBody) {` condition

Reply:
I have added unit test proposed by you and changed code a bit